Resources/Prototypes/Catalog/Fills/Boxes/general.yml

```diff
@@ -427,6 +427,42 @@
   - type: RadiationBlockingContainer
     resistance: 2
 
+- type: entity
+  name: candle box
+  parent: BoxCardboard
+  id: BoxCandle
+  components:
+  - type: Item
+    size: 60
+  - type: Sprite
+    layers:
+      - state: box
+      - state: candle
+  - type: Storage
+    capacity: 60
+  - type: StorageFill
+    contents:
+      - id: Candle
+        amount: 3
+      - id: CandleBlue
+        amount: 3
+      - id: CandleRed
+        amount: 3
+      - id: CandleGreen
+        amount: 3
+      - id: CandlePurple
+        amount: 3
+      - id: CandleSmall
+        amount: 5
+      - id: CandleBlueSmall
+        amount: 5
+      - id: CandleRedSmall
+        amount: 5
+      - id: CandleGreenSmall
+        amount: 5
+      - id: CandlePurpleSmall
+        amount: 5
```

Comment:
40 candles is a bit too many entities to dump in one box, this should ideally be split up.

Reply:
![image](https://github.com/space-wizards/space-station-14/assets/96445749/294c1711-a955-4ae3-8c7f-09126b061c8e)
Splitted